pkgs/development/libraries/pmdk/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, pkgconfig, autoconf, automake, doxygen, graphviz, pandoc
+}:
+
+stdenv.mkDerivation rec {
+       name="pmdk-1.4.1";
+       src = fetchFromGitHub {
+           owner = "pmem";
+           repo = "pmdk";
+           rev = "1.4.1";
+           sha256 = "0ylif2ws1zz5dcfgidmm7nyhmvshclcas22gyx7zhyrkq2zs5npc";
+       };
+
+       buildInputs = [
+          pkgconfig
+          autoconf
+          automake
+          doxygen
```

Comment:
pkgconfig autoconf automake and doxygen are probably only needed at build time so should be `nativeBuildInputs`